doc/langref.html.in

```diff
@@ -2990,6 +2990,12 @@ pub fn main() anyerror!void {
       {#header_close#}
 
       {#header_open|Slices#}
+      <p>
+      A slice is a pointer and a length. The difference between an array and
+      a slice is that the array's length is part of the type and known at
+      compile-time, whereas the slice's length is known at runtime.
+      Both can be accessed with the `len` field.
+      </p>
```

Comment:
This is the exact same description found in comments below in the code example. Why do we need this twice?